openj9.test.jlm/build.xml

```diff
@@ -1,111 +1,117 @@
+		must be set on the command line - e.g. -Dstf_root=/my_stf_build_dir
+	-->
+	<property name="stf_root" location="${source_root}/../stf"/>
+	<property name="openj9_systemtest_root" location="${source_root}/../openj9-systemtest"/>
+
+	<!-- Import settings used by multiple projects.  -->
+	<import file="${stf_root}/stf.build/include/top.xml"/>
+
+	<!-- For modularity you need a directory above the package structure to hold the module.java file -->
+	<property name="openj9_test_jlm_module" value="test.jlm.ibm" />
+	<property name="openj9_test_jlm_src_dir" value="src/${openj9_test_jlm_module}" />
+	<property name="openj9_test_jlm_bin_dir" value="bin" />
+
+	<property name="openj9_test_jlm_jar_file" value="${openj9_test_jlm_bin_dir}/${openj9_test_jlm_module}.jar" />
+	
+	<path id="openjdk.test.jlm.classpath">
+		<pathelement location="${openj9_systemtest_root}/../openjdk-systemtest/openjdk.test.jlm/bin/test.jlm.jar" />
+	</path>
+
+	<!-- We need junit and stf to compile this project. -->
+	<path id="project.class.path">
+		<path refid="junit.class.path" />
+		<path refid="stf.class.path" />
+		<path refid="openjdk.test.jlm.classpath" />
+	</path>
+
+	<target name="build" depends="check-prereqs, build-no-natives, build-natives">
+	</target>
+
+	<target name="build-no-natives" depends="check-prereqs, build-archives">
+	</target>
+
+	<target name="build-natives">
+	</target>
+
+	<target name="build-archives" depends="build-jar">
+	</target>
+
+	<target name="build-jar" depends="build-java, create-bin-dir">
+		<jar destfile="${openj9_test_jlm_jar_file}">
+			<fileset dir="${openj9_test_jlm_bin_dir}" includes="**/*.class" />
+		</jar>
+	</target>
+
+	<target name="build-java" depends="check-prereqs, create-bin-dir">
+		 <!--
+		 	The Ant javac task only checks time dependencies between .java files and their .class files,
+			so fails to recompile in situations such as the signatures of a dependent method changing.
+			The depend target checks the dependencies and deletes any .class files older than the files
+			which depend on them, thereby ensuring recompilation.
+		-->
+		<!--
+		Commented out because depend appears to be evaluating class files as always out of date with jdk11 (at 28/03/2019)
+		<depend srcdir="${openj9_test_jlm_src_dir}" destdir="${openj9_test_jlm_bin_dir}" classpathref="project.class.path">
+			<include name="**/*.java"/>
+		</depend>
+		-->
+		<javac srcdir="${openj9_test_jlm_src_dir}"
+			   destdir="${openj9_test_jlm_bin_dir}"
+			   fork="true"
+			   executable="${java_compiler}"
+			   debug="true"
+			   classpathref="project.class.path"
+			   encoding="${src-encoding}"
+			   includeantruntime="false"
+			   failonerror="true">
+			<include name="**/*.java"/>
+		</javac>
+	</target>
+
+	<target name="create-bin-dir">
+		<mkdir dir="${openj9_test_jlm_bin_dir}"/>
+	</target>
+
+	<target name="clean">
+		<delete dir="${openj9_test_jlm_bin_dir}"/>
+	</target>
+
+</project>
```

Comment:
Can we go over this file's changes and ensure the PR contains only the differences for changes made? Looks like there is a format difference between the local and remote file - which caused the entire file to show up as change

Reply:
A number of the test.jlm files had Windows CRLF rather than unix line feed so I fixed them - but that caused the PR checker to insist on updating the copyright year.